packages/circuit-ui/util/test-utils.tsx

```diff
@@ -49,13 +49,16 @@ const WithProviders: FunctionComponent<PropsWithChildren<unknown>> = ({
 const render: RenderFn<RenderResult> = (component, options: RenderOptions) =>
   renderTest(component, { wrapper: WithProviders, ...options });
 /**
- * FIXME: `renderToHtml` should be removed and replaced in a11y tests by
- * const { container } = render(<Component />);
+ * @deprecated renderToHtml is deprecated. Use `axe(screen)` in accessibility
+ * tests instead.
  */
 const renderToHtml: RenderFn<HTMLElement> = (component) => {
   const { container } = render(component);
   return container;
 };
+/**
+ * @deprecated create is deprecated. Use render instead.
+ */
```

Comment:
Deprecated these two render functions because I find that they introduce unnecessary complexity and inconsistencies in the repository. I'd like to simplify it to a single `render` method, which should work in (almost?) all cases. Any exceptions can be handled on a spec-by-spec basis

The only real impact is that snapshots would be wrapped by a `div`. I don't think that this is actually an issue, and it will at least make behavior across all test renders consistent 🙂